### ISSUE TITLE
Multiple -P Not Allowed

### ISSUE THREAD

**Issue by jesseherrera** [ruffle-rs/ruffle#5252]
Hello, I just tried the latest version of and I am getting an error now when I have multiple flashvars using -P more than once. I get the following error on Windows 10:

error: The argument '-P <PARAMETERS>...' requires 1 values, but 2 were provided

This happened when launching ruffle from the command line. It can be reproduced trying to run the command listed in the help.

ruffle.exe -Pkey=value -Pfoo=bar

**Comment by parkerlreed** [comment]
What happens if you try

-Pkey=value,foo=bar

**Comment by jesseherrera** [comment]
Hiya, It doesn't display an error which is good. However, my flash file just displays a loading icon so it doesn't seem to be recognizing the parameters the way an old version of Ruffle did when I was using multiple -P on the command line for my flashvars.  I have a bunch of graphs built using http://maani.us/xml_charts/ and it was semi working with an older version of ruffle. I updated to the latest version and got the error I reported earlier.
